packages/docs/src/styles/index.scss

```diff
@@ -1,29 +1,30 @@
-$font-path: '@cmsgov/ds-medicare-gov/dist/fonts';
+$font-path: '@cmsgov/ds-medicare-gov/src/fonts';
 @import '@cmsgov/ds-medicare-gov/src/styles/fonts';
 
-// importing core styles from core index.scss files
-@import 'theme-variables/_core-scss-to-css.map.scss';
-@import 'theme-variables/_core-theme.css';
-@import 'theme-variables/_core-components-scss-to-css.map.scss';
-@import 'theme-variables/_core-components-theme.css';
-// brand themes will use core variables if the theme does not explicitly define that variable
-@import 'theme-variables/_healthcare-scss-to-css.map.scss';
-@import 'theme-variables/_healthcare-theme.css';
-@import 'theme-variables/_healthcare-components-scss-to-css.map.scss';
-@import 'theme-variables/_healthcare-components-theme.css';
-@import 'theme-variables/_medicare-scss-to-css.map.scss';
-@import 'theme-variables/_medicare-theme.css';
-@import 'theme-variables/_medicare-components-scss-to-css.map.scss';
-@import 'theme-variables/_medicare-components-theme.css';
+// // importing core styles from core index.scss files
+// @import 'theme-variables/_core-scss-to-css.map.scss';
+// @import 'theme-variables/_core-theme.css';
+// @import 'theme-variables/_core-components-scss-to-css.map.scss';
+// @import 'theme-variables/_core-components-theme.css';
+// // brand themes will use core variables if the theme does not explicitly define that variable
+// @import 'theme-variables/_healthcare-scss-to-css.map.scss';
+// @import 'theme-variables/_healthcare-theme.css';
+// @import 'theme-variables/_healthcare-components-scss-to-css.map.scss';
+// @import 'theme-variables/_healthcare-components-theme.css';
+// @import 'theme-variables/_medicare-scss-to-css.map.scss';
+// @import 'theme-variables/_medicare-theme.css';
+// @import 'theme-variables/_medicare-components-scss-to-css.map.scss';
+// @import 'theme-variables/_medicare-components-theme.css';
 
 // @styles is aliased in gatsby-node.js
+@import '@cmsgov/design-system/src/styles/layout';
 @import '@styles/index.scss';
 
-[data-theme='healthcare'] {
-  // Have to set the image path so that the error label's svg can be found
-  $image-path: '@cmsgov/ds-healthcare-gov/src/images';
-  @import '@cmsgov/ds-healthcare-gov/src/styles/components/index';
-}
+// [data-theme='healthcare'] {
+//   // Have to set the image path so that the error label's svg can be found
+//   $image-path: '@cmsgov/ds-healthcare-gov/src/images';
+//   @import '@cmsgov/ds-healthcare-gov/src/styles/index';
+// }
```

Comment:
Checking my understanding: We don't need to worry about including healthcare- or medicare-specific CSS in the doc site CSS because we don't directly embed any of those components and instead rely on the storybook styles being complete, right?

Reply:
Just the fonts, I will look closer at fonts in this regard tomorrow.